### PR TITLE
Evgskv/colab example

### PR DESCRIPTION
**Pull request by EvgSkv**
Adding CoLab with installation of the framework.

### CODE REVIEW

**Review by huangxichen1 — APPROVED** [review]
Thanks, Evgeny!

File: src/estimators/liquid_legions.py [hunk 1]
```diff
@@ -103,12 +103,15 @@ def legions_expectation(self, t):
     """Expected number of legionaries activated for cardinality."""
     cache_key = (self.a, self.m, t)
     if cache_key not in self.memoized_cardinality:
-      a = self.a
-      z = 1 - (-special.expi(- a * t / (numpy.exp(a) - 1))
-               +special.expi(-a * numpy.exp(a) * t / (numpy.exp(a) - 1))) / a
-      r = numpy.where(t == 0, 0, z)
-      self.memoized_cardinality[cache_key] = r
-    return self.memoized_cardinality[cache_key]
+      if t == 0:
```
Comment (kungfucraig) [inline src/estimators/liquid_legions.py:106]: This should be in a different change.

**Review by kungfucraig — CHANGES_REQUESTED** [review]

File: src/estimators/liquid_legions.py (hunk 1 above)
Comment (EvgSkv) [reply src/estimators/liquid_legions.py]: Done.

**Review by kungfucraig — APPROVED** [review]

**Comment by EvgSkv** [comment]
The scope was changed just to add documentation.

**Pull request merged**